RecoJets/JetProducers/python/puJetIDParams_cfi.py

```diff
@@ -52,6 +52,28 @@
                 Pt3050_Loose   = cms.vdouble(-0.15,-0.26,-0.16,-0.16),
                 )
 
+full_74x_chs_wp  = cms.PSet(
```

Comment:
this looks like a full copy of what's in 
PileupJetIDCutParams_cfi
why not just

``` from RecoJets.JetProducers.PileupJetIDCutParams_cfi import full_74x_chs_wp ```
it's unclear overall why there are two files at all, just to confuse everyone?